packages/api/src/entity/library_item.ts

```diff
@@ -196,4 +196,28 @@ export class LibraryItem {
 
   @Column('text', { nullable: true })
   recommenderNames?: string[] | null
+
+  @Column('timestamptz')
+  hiddenAt?: Date | null
+
+  @Column('timestamptz')
+  sharedAt?: Date | null
+
+  @Column('text')
+  sharedBy?: string | null
+
+  @Column('jsonb')
+  links?: any | null
+
+  @Column('text')
+  previewContent?: string | null
```

Comment:
I wonder if we need a `previewContentType` or is it always HTML from feeds?

Reply:
I think we need `previewContentType` because I'm not sure if it is always HTML